wiki/Ranking_Criteria/en.md

```diff
@@ -106,7 +106,7 @@ Changes to the rules and guidelines in this document are proposed and discussed
 
 #### Romanisation
 
-- **An artist's name is to be romanised in the order it is printed in the unicode field, unless an alternative order for the romanised name is provided by the artist.**
+- **An artist's name must be romanised in the order it is printed in the unicode field if there is no official romanisation or translation available.** If an official romanisation is available, either option may be used in accordance with the relevant allowance.
```

Comment:
what does "relevant allowance" mean? i can't understand this.

Reply:
Same one changed in the PR, line 149 which allows people to choose if they have good reason. Ideally it would be cross linked but not sure on how that would be done.

Reply:
I think peppy refers to the complexity of that sentence and that it could be simplified (I honestly barely understand it either)

Reply:
Reworded, please follow up if still unclear.